zipline/assets/asset_db_schema.py

```diff
@@ -162,6 +163,29 @@
     sa.Column('asset_type', sa.Text),
 )
 
+supplementary_mappings = sa.Table(
+    'supplementary_mappings',
+    metadata,
+    sa.Column(
+        'id',
```

Comment:
My instinct is to make a compound pk on those unique columns instead of adding the surrogate key.  Not sure if there's a performance impact...